packages/manager/modules/support/src/ticket/index.js

```diff
@@ -0,0 +1,29 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+import { state } from './ticket.routing';
+
+const moduleName = 'ovhManagerSupportTicketLazyLoading';
+
+angular
+  .module(moduleName, [
+    'ovhManagerCore',
+    'oc.lazyLoad',
+    'ui.router',
+  ])
+  .config(/* @ngInject */ ($stateProvider) => {
+    $stateProvider.state(`${state.name}.**`, {
```

Comment:
Why do you use constants for state name ?
IMO it make the code more complicated to read.